app/components/Story.js

```diff
@@ -22,48 +22,62 @@ class Story extends Component {
     const ds = new ViewPager.DataSource({pageHasChanged: (r1, r2) => r1.text !== r2.text});
     this.state = {
       data: ds.cloneWithPages(tempData),
+      isSplit: this.props.viewMode === 'pair'
     };
 
-    this.onClick = this.onClick.bind(this);
+    this.onClickBack = this.onClickBack.bind(this);
+    this.onClickToggle = this.onClickToggle.bind(this);
     this._renderPage = this._renderPage.bind(this);
   }
 
-  onClick() {
+  onClickBack() {
     this.props.handleNavigate({
       type: 'pop',
       route: {key: 'pop'},
     });
   }
 
+  onClickToggle() {
+    this.setState({
+      isSplit: !this.state.isSplit
+    });
+  }
+
   _renderPage(data) {
-    if (this.props.viewMode === 'single') {
-      return (
-        <View style={styles.container}>
-          <View style={styles.card}>
-            <Text style={styles.text}>
-              {data.topText}
-            </Text>
-          </View>
-        </View>
-      );
+    if (this.state.isSplit) {
+      return <StoryCard topText={data.topText} bottomText={data.bottomText} />;
     }
 
-    return <StoryCard topText={data.topText} bottomText={data.bottomText} />;
+    return (
+      <View style={styles.container}>
+        <View style={styles.card}>
```

Comment:
this is going to be gone in my next pass, since we're putting everything through StoryCard